packages/actions/src/eth/ethSendRawTransaction.spec.ts

```
@@ -0,0 +1,146 @@
+    const handler = ethSendRawTransactionHandler(client)
+
+    // Create mock KZG commitments (32 bytes each, starting with 0x01)
+    const mockKZGCommitment1 = new Uint8Array([0x01, ...new Uint8Array(31).fill(1)])
+    const mockKZGCommitment2 = new Uint8Array([0x01, ...new Uint8Array(31).fill(2)])
+
+    const blobTx = BlobEIP4844Transaction.fromTxData({
+      nonce: '0x00',
+      maxFeePerGas: '0x09184e72a000',
+      maxPriorityFeePerGas: '0x09184e72a000',
+      gasLimit: '0x2710',
+      to: createAddress('0x' + '42'.repeat(20)),
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)